modules/StudentManagemnt.py

```diff
@@ -4,21 +4,26 @@
 OurStudents = []
 
 def add_student(student):
+    global OurStudents
```

Comment:
Global variable is a disaster 
Ask ChatGPT how to improve this
Please try to limit using global variables in general